test-files/cases/attribute/unifying/highway-2888/Expected.osm

```diff
@@ -4709,6 +4709,55 @@
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-1"/>
     </node>
+    <way visible="true" id="-1000" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-111"/>
+        <nd ref="-109"/>
+        <nd ref="-108"/>
+        <nd ref="-107"/>
+        <nd ref="-135"/>
+        <nd ref="-1035"/>
+        <tag k="length" v="141.8"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-1000"/>
+    </way>
+    <way visible="true" id="-984" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-1107"/>
+        <nd ref="-1106"/>
+        <tag k="length" v="54.3"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-984"/>
+    </way>
+    <way visible="true" id="-982" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-1093"/>
+        <nd ref="-1050"/>
+        <tag k="length" v="279.6"/>
+        <tag k="source:datetime" v="2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="road"/>
+        <tag k="hoot:status" v="1"/>
+        <tag k="hoot:id" v="-982"/>
+    </way>
+    <way visible="true" id="-979" timestamp="2019-01-24T18:58:28Z" version="1">
+        <nd ref="-840"/>
+        <nd ref="-824"/>
+        <nd ref="-823"/>
+        <tag k="source:imagery:datetime" v="2019-01-04 06:01:44"/>
+        <tag k="source:imagery:sensor" v="WV03_VNIR"/>
+        <tag k="length" v="300.2"/>
+        <tag k="security:classification" v="UNCLASSIFIED"/>
+        <tag k="source:imagery" v="DigitalGlobe"/>
+        <tag k="source" v="Bing"/>
+        <tag k="name" v="Gulhazaaru Goalhi"/>
+        <tag k="source:imagery:layerName" v="2019-01-04 - WV03_VNIR"/>
+        <tag k="source:datetime" v="2014-11-28T17:29:47.000Z;2019-01-24T18:58:28Z"/>
+        <tag k="highway" v="residential"/>
+        <tag k="source:imagery:id" v="5ce2e89d91bbea89ecf6dbf5915a50e2"/>
+        <tag k="hoot:status" v="3"/>
+        <tag k="hoot:id" v="-979"/>
+    </way>
```

Comment:
Way `-979` is actually a good one in my mind because that vertical subline matches with OSM data and gets a different name instead of continuing that way around the corner.
![2888-way-979](https://user-images.githubusercontent.com/13385275/53367284-57161800-390b-11e9-8883-70128ba309d6.png)

Reply:
Definitely.  This took bad conflation output and made it better.  Due to the difference in geometries between ref and sec, this is probably about as good as this section is going to get.